vNext/src/NuGet/Microsoft.Orleans.OrleansAzureUtils.nuspec

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansAzureUtils</id>
+    <version>$version$</version>
+    <title>Microsoft Orleans Azure Utilities</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Azure Utilities Library of Microsoft Orleans - OrleansAzureUtils.dll
+    </summary>
+    <description>
+      Library of utility types for Microsoft Azure of Microsoft Orleans.
```

Comment:
maybe something like "Support for hosting Orleans on Microsoft Azure.", similarly, the summary should say something like "Azure support for Microsoft Orleans"